1-Docker/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+services:
+  headnode:
+    image: dahachm/hadoop:headnode
```

Comment:
Tags should be used to track versions of one image (ex. [Jenkins DockerHub](https://hub.docker.com/_/jenkins?tab=tags&page=1&ordering=last_updated) ), not to distinguish between headnode and worker.
There should be two separate images, i.e.: dahachm/hadoop_headnode:1.2 (or latest) and ahachm/hadoop_worker:1.15 (or latest)

Reply:
Okay, was guessing is it right to put them in the same repo with different tags

The reason why I put them together was about the fact that thay are in the same task and they both are hadoop containers

Should I move one to another repo?

Reply:
You can do it, if you'll have spare time, but this is not necessary to pass hometask =)